composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/SpotExplorerView.kt

```diff
@@ -56,5 +61,28 @@ fun SpotExplorerView(mapViewModel: MapViewModel) {
                 Box(modifier = Modifier.fillMaxSize().background(Color.Transparent))
             }
         }
+
+        AnimatedContent(
+            targetState = showFiltering,
+            transitionSpec = {
+                slideInHorizontally(initialOffsetX = { fullWidth -> fullWidth }) + fadeIn() togetherWith
+                        slideOutHorizontally(targetOffsetX = { fullWidth -> fullWidth }) + fadeOut() using
+                        SizeTransform(clip = false)
+            }
+        ) {
```

Comment:
If you make this:

```
) { show ->
      if (show)
```

This transition won't "double bounce".

Reply:
smart smart update this